### PR TITLE
Feature functional

### PR DESCRIPTION
**Pull request by SotoArmando**
Ready for review :D

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Odin Restaurant</title>
+    <link href="css/main.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700,900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div id="content">
+        <nav>
```
Comment (JamesOkunlade) [inline dist/index.html:15]: I didn't expect to see any code here other than an empty `<div id="content"> </div>` you will use JavaScript in your `index.js` file to append views to the `div`. All the coding will be done in your src folder and `webpack`  after runing `npm run build` will compile it and generate it into the `main.js` in your `dist`.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+console.log("am a frog not a virus trust me ;)")
```
Comment (JamesOkunlade) [inline src/index.js]: `Console.log` kills the performance of an app please remove it.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+const space = [
+    document.querySelector("first-content"),
+    document.querySelector("second-content"),
+    document.querySelector("third-content")
+];
+
+console.log(space);
```
Comment (JamesOkunlade) [inline src/index.js]: Remove this too.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hello @SotoArmando Nice work 👏 on the Restaurant page. Quick things you'll need to fix though. I promise this won't be tedious 😁 

- Setup stickler for this repo and add linter to your project. Also ensure to make the linter checks pass.
- Write a more descriptive README including the tech stack you're using, a link to the live version if available and a link to the author i.e you 😁 
-The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack. Therefore, it important you don't leave you code in the `/dist/index.html` like you have done, but instead in modules in your src using JavaScript to append each of them to the `div id="content">` in your `/dist/index.html`.

Please fix those and request for re-review. 

**Status: `Changes required`**

Happy coding!

File: src/index.js (hunk 2 above)
Comment (SotoArmando) [reply src/index.js]: Do you mean this? i was kind sure was that something like this. @JamesOkunlade
Screenshot 2019-09-20 at 16 59 40

**Review by SotoArmando — COMMENTED** [review]
Test comment

**Review by AdnanAfsari — CHANGES_REQUESTED** [review]
Hello @SotoArmando ✋ 

Nice work but you are yet to make some changes the previous reviewer requested 😸.

He wanted you to use modular code which is required and goals for this project.Please check his last review comment.

- [x] It appends HTML elements to the <div> with id=content when the page is loaded (Use document.createElement and document.appendChild

- [x] It features a tabbed menu. When a tab is clicked the content is replaced

- [x] The contents of each tab are stored in a module.

- [x] Each module exports a function that creates a <div> element, adds the appropriate content and styles to that element and then appends it to the DOM.
`
```javascript
const myModule = (parentNode) => {
    const node = document.createElement('div');
    // Add whichever sub-nodes, ids and style is required

    parentNode.appendChild(node);
};

export { myModule as default };
````

- [x] Can we be more descriptive with our README?? Please check some projects readme for inspiration.

**Friendly reminder**: You should use Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

Please submit for another review after making those changes :)

Happy Coding

**Review by JamesOkunlade — APPROVED** [review]
### Good job! It's a 👍 for me.

**Please go ahead and merge.**

**Status: `Approved`**

**Good luck with your future projects 😎**

**Pull request merged**